### PR TITLE
add GoogleAutocomplete

### PR DESCRIPTION
**Pull request by djamesfar**
added emilianotisato/nova-google-autocomplete-field.  Cannot test because of seo required addresses version conflicting...

### CODE REVIEW

File: src/Nova/DomesticAddress.php [hunk 1]
```diff
@@ -20,29 +22,35 @@ class DomesticAddress extends BaseResource
...
             }),
             Text::make('City', 'city.id', function () {
                 return $this->city->title;
             })->sortable(),
             Text::make('Zip', 'zip.code', function () {
                 return $this->zip->code;
-            })->sortable(),
+            })->sortable(),*/
         ]);
     }
```
Comment (drewroberts) [inline src/Nova/DomesticAddress.php:55]: This shouldn't go into the index fields. Just the regular fields in the nova resource when creating and updating the resource.

Comment (djamesfar) [reply src/Nova/DomesticAddress.php]: Note that tests are failing probably because GoogleAutoComplete is expecting the api key in the .env file

Comment (drewroberts) [reply src/Nova/DomesticAddress.php]: Yes, I can include it in the GitHub organization secrets

**Comment by drewroberts** [comment]
@pdbreen Could you update this package to use an organization GitHub Secret similar to how we use them for Nova:

https://github.com/tipoff/addresses/blob/5f9882c46fc0f89063c625eae5b76582158ef12e/.github/workflows/run-tests.yml#L40-L41

Let me know what name I should use for the Secret and I'll add it to the Tipoff organization. I'll also send it to you via email so you can test the package locally.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -54,4 +54,6 @@ jobs:
           composer update --${{ matrix.stability }} --prefer-dist --no-interaction
 
       - name: Execute tests
+        env:
+          ADDRESS_AUTOCOMPLETE_API_KEY: ${{ secrets.AddressAutocompleteApiKey }}
```
[the remark this answers is not in the document]
Comment (drewroberts) [reply .github/workflows/run-tests.yml:58]: It looks like it does the secrets as all capital letters.

[the remark this answers is not in the document]
Comment (drewroberts) [reply .github/workflows/run-tests.yml:58]: Nevermind, it doesn't matter how you reference them. It is case insensitive.

**Comment by drewroberts** [comment]
@pdbreen Could you get this PR to green? I added the secret in the GitHub organization

File: src/Nova/DomesticAddress.php [hunk 3]
```diff
@@ -54,9 +54,9 @@ public function fields(Request $request)
             GoogleAutocomplete::make('Address')
                 ->countries('US')
                 ->withValues(['street_number.long_name','locality.long_name','postal_code.short_name','administrative_area_level_1.short_name'])->onlyOnForms(),
-            AddressMetadata::make(['Address Line 1'])->fromValue('street_number')->onlyOnForms(),
-            AddressMetadata::make(['City'])->fromValue('locality')->onlyOnForms(),
-            AddressMetadata::make(['Zip'])->fromValue('postal_code')->onlyOnForms(),
+            AddressMetadata::make('Address Line 1')->fromValue('street_number')->onlyOnForms(),
+            AddressMetadata::make('City')->fromValue('locality')->onlyOnForms(),
+            AddressMetadata::make('Zip')->fromValue('postal_code')->onlyOnForms(),
```
Comment (pdbreen) [inline src/Nova/DomesticAddress.php:59]: @drewroberts this fix for the incorrect method call will fix the tests, but I don't know if the control will actually work.  With the original code, there's no way this form was ever called up and tested manually.

Comment (drewroberts) [reply src/Nova/DomesticAddress.php:59]: Thank you. I'll have them do some manual tests on it today.

Comment (drewroberts) [reply src/Nova/DomesticAddress.php:59]: CC: @joshtorres & @tipoff/omnia

**Review by drewroberts — APPROVED** [review]
Thank you! 🔱

We need to test this out manually in Nova and make some adjustments to get it more flexible to accepting more types of addresses. @tipoff/omnia This is our top priority for today as it is still holding up development on a few of the other packages.

**Pull request merged**